doc/INSTALL_ru-RU.md

```diff
@@ -0,0 +1,101 @@
+
+## Установить из бинарных файлов
+
+Выберите версию в разделе [Releases](https://github.com/cpeditor/cpeditor/releases), затем загрузите свой двоичный файл.
+
+### Windows
+
+Загрузите `setup.exe` в активе и установите его. Также может потребоваться установить [распространяемый пакет Microsoft Visual C++ для Visual Studio 2015, 2017 и 2019](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads).
+
+### Linux
+
+Загрузите `AppImage` в активы, добавьте к нему разрешение на выполнение (`chmod +x <AppImagePath>`), а затем запустите его.
+
+### MacOS
+
+Загрузите приложение `.dmg` файл в активах и установите его.
+
+Возможно, Вам потребуется нажать кнопку "Да", чтобы получить доступ к файловой системе несколько раз. Apple может сообщить, что это приложение не исходит от признанного разработчика и не позволяет вам запускать приложение. Пожалуйста, добавьте исключение в Редакторе из `Settings -> Security -> General`, и разрешить CPEditor `выполнять в любом случае`.
+
+## Сборка из исходного кода
+
+1. Клонируйте этот репозиторий и подмодули:
+
+	```sh
+	git clone --recurse-submodules https://github.com/cpeditor/cpeditor.git
+	cd cpeditor
+	```
+
+2. Установите [Qt](https://www.qt.io/download) (5.12 или выше) и [CMake](https://cmake.org/download/) (3.12 или выше).
+   - На некоторые дистрибутивы Linux и MacOS, вы можете установить из вашего менеджера пакетов. Например, `sudo pacman-S qt5` на Arch Linux, `brew install qt5` на Mac OS.
+   - Вы также можете скачать [автономный установщик](https://www.qt.io/offline-installers), или скачать с [зеркал](https://download.qt.io/static/mirrorlist/). Путь от корня зеркала должен быть похож на `/qt/official_releases/qt/5.14/5.14.1/qt-opensource-<platform>-5.14.1.<suffix>` (или другие версии).
+   - Вы также можете использовать [aqtinstall](https://github.com/miurahr/aqtinstall) чтобы установить Qt.
+
+3. Если CMake не может найти путь установки Qt, вы должны установить переменную окружения: `CMAKE_PREFIX_PATH=%QtPath% / %QtVersion% / %Compiler% / lib / cmake`. Например, на Mac вы можете запустить что-то вроде `export CMAKE_PREFIX_PATH="/usr/local/Cellar/qt/5.14.1`.
+
+4. Выполните следующие команды:
+
+	- Linux/MacOS:
+
+		```sh
+		mkdir build
+		cd build
+		cmake .. -DCMAKE_BUILD_TYPE=Release
+		cmake --build .
+		```
+
+	- Windows:
+
+		```bat
+		mkdir build
+		cd build
+		cmake ..
+		cmake --build . --config Release
+		```
+
+В Linux вы получите `build/cpeditor`.
+
+На Mac вы получите `build/cpeditor.app`.
+
+В Windows вы получите `build/cpeditor.exe`, или `build/Release/cpeditor.exe`. Возможно, вам потребуется собрать необходимые библиотеки DLL. Если вы установили редактор CP с помощью `setup.exe`, вы можете скопировать библиотеки DLL, из которых установлен редактор CP (или скопировать исполняемый файл в путь установки).
+
+## Использовать Артефакты
```

Comment:
"Artifacts" shouldn't be translated. Remember to change the following paragraphs as well.